Assets/Scripts/Menu/CueCardMenu.cs

```diff
@@ -72,4 +73,15 @@ public static void SaveCueCards()
         bf.Serialize(file, cardmap);
         file.Close();
     }
+
+    public void RemoveReturnCharacters()
+    {
+        if (questionInput.text.Contains("\n")) {
```

Comment:
I think you should truncate all white space because I can make my question/answer text all spaces and it will work.